sdk/loadtesting/azure-developer-loadtesting/MANIFEST.in

```diff
@@ -1,8 +1,7 @@
-include _meta.json
-recursive-include tests *.py *.json
-recursive-include samples *.py *.md
 include *.md
-include azure/__init__.py
-include azure/developer/__init__.py
 include LICENSE
 include azure/developer/loadtesting/py.typed
+recursive-include tests *.py
+recursive-include samples *.py *.md
+include azure/__init__.py
+include azure/developer/__init__.py
```

Comment:
@NiveditJain Did you change `MAINFEST.in`, `setup.py` manually? If so, your change will be overwrote by autorest.python next time.

Reply:
No, these was a result of running autorest